website/content/blog/2024-12-01-a-year-in-review.md

```diff
@@ -0,0 +1,116 @@
+  * [A better 2D onboarding tutorial](https://github.com/MonoGame/MonoGame/issues/8317)  
+  * [A better 3D onboarding tutorial](https://github.com/MonoGame/MonoGame/issues/8318)
+  * [2D StartKit](https://github.com/MonoGame/MonoGame/pull/8275)
+
+* #### Bug Fixes and Enhancements
+
+  * Re-enabled the Unit Tests for PR’s.  
+  * [Upgrade Mac Editor to net8.0-macos. Upgrade Eto.Forms](https://github.com/MonoGame/MonoGame/pull/8505)   
+  * Fix Issues with OpenAL Panning on DesktopGL and Mobile  [#8480](https://github.com/MonoGame/MonoGame/pull/8480) and [#8466](https://github.com/MonoGame/MonoGame/pull/8466)
+  * [Detect GamePad/Keyboard events on Android](https://github.com/MonoGame/MonoGame/pull/8465)
+  * [Remove string allocations when calling GamePad.GetCapabilities](https://github.com/MonoGame/MonoGame/pull/8453)
+  * [Content Pipeline now works on M1/M2 Macs](https://github.com/MonoGame/MonoGame/pull/8570)
+  * [Fix Android MediaPlayer Song implementation](https://github.com/MonoGame/MonoGame/pull/8583)
+  * [Update MonoGame.Library.OpenAL to 1.23.1.10](https://github.com/MonoGame/MonoGame/pull/8560)
+  * [Fix Sprite font multi byte parsing](https://github.com/MonoGame/MonoGame/pull/8554)
+  * [Added Vibration and Caps to Native GamePad API](https://github.com/MonoGame/MonoGame/pull/8520)
+  * [Fix MacOS Content directory location](https://github.com/MonoGame/MonoGame/pull/8479)
+  * [Detect GamePad/Keyboard events on Android](https://github.com/MonoGame/MonoGame/pull/8465)
+
+* #### Games Released (that we know of)
+
+  * [Horticular](https://store.steampowered.com/app/1928540/Horticular/) by inDirection Games
+  * [Breeze & Freeze](https://msiebenmann.itch.io/breeze-and-freeze) by msiebenmann
+  * [Tic Tac Dough](https://ms00.itch.io/tic-tac-dough) by MS00
+  * [Ninja Cat Remewstered](https://pixelshock.itch.io/ninja-cat-remewstered) by Pixel Shock
+  * [Smack 'n' Snatch](https://skyyyla.itch.io/smack-n-snatch) by Skyla
+  * [FishGrid](https://shmellyorc.itch.io/fishgrid) by Shmellyorc
+  * [Axe Attacks](https://serork.itch.io/axe-attacks) by Serork
+  * [Mochi's Escape](https://anonames-lair.itch.io/mochi-escape) by Anonames
+  * [Armored Crucible: The Crimson Dunes](https://fontty.itch.io/ac-tcd) by Fontty Games
+  * [Arid Arnold](https://icefish-software.itch.io/arid-arnold) by IceFish Software
+  * [Dungeon Adventure Gang](https://store.steampowered.com/app/2147690/Dungeon_Adventure_Gang/) by DAG Team
+  * [Cranky Chibi Cthulhu](https://little-tlaloc-interactive.itch.io/cranky-chibi-cthulhu) by Little Tlaloc Interactive. Uses [FlatRedBall](https://flatredball.com/) and [Gum](http://docs.flatredball.com/gum/)
+  * [BattleCrypt Bombers](https://store.steampowered.com/app/2188930/BattleCrypt_Bombers/) by Narfox LLC. Uses [FlatRedBall](https://flatredball.com/) and [Gum](http://docs.flatredball.com/gum/)
+
+
+### What is coming in the next year:
+
+* #### 3.8.3 Release
```

Comment:
Are we promising DX12, Vulkan, and new effect compilers working for the specific 3.8.3 release?

Reply:
I've shifted the other elements back, so it doesn't look like they are part of the 3.8.3 release and removed the Headline tags on the 3.8.3 Release, so it now all looks like it's part of Coming in the next year.